libyul/AsmParser.cpp

```diff
@@ -70,7 +70,7 @@ Block Parser::parseBlock()
 	expectToken(Token::LBrace);
 	while (currentToken() != Token::RBrace)
 		block.statements.emplace_back(parseStatement());
-	block.location.end = currentLocation().end;
+	changeDebugDataLocationEnd(block.debugData, currentLocation().end);
```

Comment:
Can we avoid the io parameter?
```suggestion
	block.debugData = updateLocationEnd(block.debugData, currentLocation().end);
```

Reply:
Also, do we really need a "plain" `end` parameter? I would assume the second argument here always being a sourcLocation or DebugData anyway. Then it could be `updateLocationEndFrom` or something.

Reply:
I removed io parameter, and went with `SourceLocation const&` parameter instead `int`